### DISCUSSION TITLE
Feature request: select from multiple dev containers in repo

### DISCUSSION THREAD

**Discussion by diberry** [community/community#61292] [Codespaces]
### Select Topic Area

Product Feedback

### Body

My monorepo has many projects which each have their own devcontainer. When I tell ppl how to open the repo with the correct/specific container, I have to take a screenshot of the ellipsis on the menu. Can detection of multiple codespace containers and display of selection be easier to find? 

This is where I have to go right now. 

![image](https://github.com/community/community/assets/41597107/b33f419d-3a30-478a-8639-853a34d01bab)

Then select new with options

![image](https://github.com/community/community/assets/41597107/03fe0d90-0d68-4410-98f4-3befe1296dc1)

Then select Dev container configuration.

![image](https://github.com/community/community/assets/41597107/1efb383d-9a70-4551-93d8-9d13ed2771dc)

**Comment by diberry** [comment]
Also, notice I'm in the azure-samples organization so that org would paid for my usage but I really need to choose to not use their money for my code space. Right now the only choice is to use a different account not in the org.

**Reply by eljog** [reply]
Right now there’s not a way to elect to pay for a codespace personally if the org has been configured to pay for your usage.

**Comment by diberry** [comment]
Is there a keyboard shortcut to get all the way to the New with options screen?

**Comment by eljog** [comment]
@diberry 
You can create custom deep links that allows the users to directly launch a codespace with a defined configuration. 
[Here is a detailed documentation](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/setting-up-your-repository/facilitating-quick-creation-and-resumption-of-codespaces#configuring-more-options) of how you use the "Share a deep link" feature.
